recipe/meta.yaml

```diff
@@ -1,6 +1,7 @@
 {% set name = "ocp" %}
 {% set version = "7.6.3.0" %}
 {% set occt_ver = "7.6.3" %}
+{% set py_version = "3.11" %}
```

Comment:
This is currently only building for Python 3.11, we would want to open this up to have the complete build matrix again.

Reply:
Done